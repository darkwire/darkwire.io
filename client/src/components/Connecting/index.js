import React, { Component } from 'react'

export default class Connecting extends Component {
  render() {
    return (
      <div>
        Please wait while we secure a connection to Darkwire...
      </div>
    )
  }
}
