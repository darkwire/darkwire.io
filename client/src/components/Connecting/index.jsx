const Connecting = () => {
  return <div>Please wait while we secure a connection to Darkwire...</div>;
};

export default Connecting;
