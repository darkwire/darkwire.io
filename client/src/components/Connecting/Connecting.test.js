import React from 'react';
import { render } from '@testing-library/react';
import Connecting from '.';

test('Connecting component is displaying', async () => {
  const { asFragment } = render(<Connecting />);

  expect(asFragment()).toMatchSnapshot();
});
