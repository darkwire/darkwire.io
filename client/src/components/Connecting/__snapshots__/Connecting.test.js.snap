// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connecting component is displaying 1`] = `
<DocumentFragment>
  <div>
    Please wait while we secure a connection to Darkwire...
  </div>
</DocumentFragment>
`;
